src/pages/certifications/coding-interview-prep/algorithms/implement-selection-sort/index.md

```diff
@@ -3,8 +3,34 @@ title: Implement Selection Sort
 ---
 ## Implement Selection Sort
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/coding-interview-prep/algorithms/implement-selection-sort/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Method:
+- Selection Sort is one of the easier sorting algorithm to understand and implement.
+- This algorithm splits the array in two parts:
+  1. Sorted
+  2. Unsorted
+ - The Sorted part is at the beginning of the array and Unsorted part afterwards.
+ - Each pass, initially we assume the first element to be the smallest then we loop through the whole array and *select* the smallest element. At the end of the pass we swap smallest element to the sorted array.
+ - It has **O(n2)** time complexity.
```

Comment:
I think you meant **O(n2)** to be **O(n<sup>2</sup>)**.

Reply:
@Nirajn2311 Fixed, thanks.